Jenkinsfile

```diff
@@ -115,19 +120,25 @@ pipeline {
      }
     }
 
-    // when env.BRANCH_NAME == 'master', we will update the mesosphere:dcos-ui/latest branch
-    // when env.BRANCH_NAME =~ /(tag-name-regex)/, we will open a release PR against dcos/dcos
-    stage('Upload Build') {
+    // update the mesosphere:dcos-ui/latest branch
+    stage('Release Latest') {
       when {
-        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+        expression {
```

Comment:
On the regex detection: Because this stage is only updating the latest build, it runs on every master change (=merged PR) 

I'll explain `CREATE_RELEASE` and tag release below. ⬇️

Reply:
Makes sense.